cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -238,17 +238,21 @@ int64_t WholeStageResultIterator::spillFixedSize(int64_t size) {
       // As of now, non-zero running threads usually happens when:
       // 1. Task A spills task B;
       // 2. Task A trys to grow buffers created by task B, during which spill is requested on task A again;
-      VLOG(2) << logPrefix << "Spill is requested on a task " << task_->taskId()
-              << " that has non-zero running threads, which is not currently supported. Skipping.";
+      LOG(WARNING) << logPrefix << "Spill is requested on a task " << task_->taskId()
+                   << " that has non-zero running threads, which is not currently supported. Skipping.";
       return shrunken;
     }
     int64_t remaining = size - shrunken;
-    LOG(INFO) << logPrefix << "Trying to request spill for " << remaining << " bytes...";
-    auto* mm = memoryManager_->getMemoryManager();
+    LOG(INFO) << logPrefix << "Trying to request spill for " << velox::succinctBytes(remaining);
```

Comment:
nit: With the punctuation mark removed the log could look a bit ugly. E.g., 

```
W20240920 11:38:50.542661 20191 HashBuild.cpp:1052] Trying to request spill for 1MB
```

Reply:
done